app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name, age):
+        self.age = age
+        self.name = name
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    ppl_list = [Person(pers["name"], pers["age"]) for pers in people]
```

Comment:
Please, Do not shorten the names of the variables. It could impair the readability of the code.
It's not clear that `ppl_list` stores people. 

The same case with the `pers` variable.